instrumentation-appender-api/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.animalsniffer-conventions")
+  id("otel.jacoco-conventions")
+  id("otel.japicmp-conventions")
+  id("otel.publish-conventions")
```

Comment:
Do we plan on publishing this? I would think it would be strictly internal.

Reply:
Oh I suppose yes, since users now need to call `GlobalLogEmitterProvider.set(..)`.